File: C#/TopflytechCodec/TopflytechCodec/Entities/BleCtrlData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    class BleCtrlData : BleData
    {
        private float voltage;

        public float Voltage
        {
            get { return voltage; }
            set { voltage = value; }
        }
        private int batteryPercent;

        public int BatteryPercent
        {
            get { return batteryPercent; }
            set { batteryPercent = value; }
        }
        private float temp;

        public float Temp
        {
            get { return temp; }
            set { temp = value; }
        }

        private int ctrlStatus;
        public int CtrlStatus
        {
            get { return ctrlStatus; }
            set { ctrlStatus = value; }
        }
        

        private int online;
        public int Online
        {
            get { return online; }
            set { online = value; }
        }
        private int rssi;

        public int Rssi
        {
            get { return rssi; }
            set { rssi = value; }
        }
    }
}

File: Java/topflytechCodec/src/com/topflytech/codec/entities/BleCtrlData.java
package com.topflytech.codec.entities;


/**
 * Created by admin on 2019/8/29.
 */
public class BleCtrlData extends BleData{

    public Float getVoltage() {
        return voltage;
    }

    public void setVoltage(Float voltage) {
        this.voltage = voltage;
    }

    public Integer getBatteryPercent() {
        return batteryPercent;
    }

    public void setBatteryPercent(Integer batteryPercent) {
        this.batteryPercent = batteryPercent;
    }

    public Float getTemp() {
        return temp;
    }

    public void setTemp(Float temp) {
        this.temp = temp;
    }


    public int getCtrlStatus() {
        return ctrlStatus;
    }

    public void setCtrlStatus(int CtrlStatus) {
        this.ctrlStatus = CtrlStatus;
    }

    public Integer getRssi() {
        return rssi;
    }

    public void setRssi(Integer rssi) {
        this.rssi = rssi;
    }

    private Float voltage;
    private Integer batteryPercent;
    private Float temp;
    private int ctrlStatus;
    private int online;

    public int getOnline() {
        return online;
    }

    public void setOnline(int online) {
        this.online = online;
    }

    private Integer rssi;


}
